packages/core/template_project/app/src/main/java/LauncherActivity.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package <%= packageId %>;
+
+import android.net.Uri;
+
+<% for(const imp of pluginManager.launcherActivity.imports) { %>
+    import <%= imp %>;
+<% } %>
+
+public class LauncherActivity
```

Comment:
We're going to want some tests that check whether the generated project actually builds.

Reply:
Yeah - ideally we could add something like this to CI

Reply:
Added tests for `AppsFlyerFeature` and `FeatureManager`. I'll start working on #38 on follow-up PR - those moving parts will make more critical to ensure the Android project build.